backend/app/api/v1/endpoints/health_check.py

```
@@ -1,19 +1,15 @@
 @router.get("")
 async def health_check(response: Response):
-    return ResponseHandler.success_response(
-        data={"status": "healthy"},
-        message="System is healthy",
-        code=200
-    )
+    """Health check endpoint."""
+    return ResponseHandler().success_response(data={"health": "healthy"}, message="Service is healthy")
 
 @router.get("/error")
 async def health_check_error(response: Response):
```

Ruff (0.14.10)

Unused function argument: `response`

(ARG001)